Resources/public/js/views/fields/ez-relation-editview.js

```diff
@@ -0,0 +1,195 @@
+                isEmpty: this._isFieldEmpty(),
+                isRequired: this.get('fieldDefinition').isRequired,
+            };
+        },
+
+        /**
+         * Tap event handler for the remove button. It resets the relation.
+         *
+         * @method _removeRelation
+         * @protected
+         * @param {EventFacade} e
+         */
+        _removeRelation: function (e) {
+            e.preventDefault();
+
+            this.set('destinationContent', null);
+            this.validate();
+            this.get('container').one('.ez-relationeditview-remove').set('disabled', true);
+        },
+
+        validate: function () {
+            if ( this.get('fieldDefinition').isRequired && this._isFieldEmpty() ){
+                this.set('errorStatus', 'This field is required');
+            } else {
+                this.set('errorStatus', false);
+            }
+        },
+
+        /**
+         * Fire the contentDiscover event with the universalDiscovery.
+         *
+         * @method _runUniversalDiscovery
+         * @protected
+         * @param {Object} e
+         */
+        _runUniversalDiscovery: function (e) {
+            e.preventDefault();
+            this.fire('contentDiscover', {
+                config: {
+                    contentDiscoveredHandler: Y.bind(this._selectRelation, this),
+                    cancelDiscoverHandler: Y.bind(this.validate, this),
+                },
+            });
+        },
+
+        /**
+         * Select the relation.
+         * Set destinationContent and destinationContentId attributes.
+         * And the error status to false.
+         *
+         * @method _selectRelation
+         * @protected
+         * @param {Object} e
+         */
+        _selectRelation: function (e) {
+            var selectedContent = e.selection.content;
+
+            this.set('errorStatus', false);
+            this.set('destinationContent', selectedContent);
+            this.get('container').one('.ez-relationeditview-remove').set('disabled', false);
+        },
+
+        /**
+         * Returns the field value.
+         *
+         * @protected
+         * @method _getFieldValue
+         * @return Object
+         */
+        _getFieldValue: function () {
+            this.validate();
+            if ( this._isFieldEmpty() ) {
+                return {destinationContentId: null};
+            } else {
+                return {destinationContentId: this.get('destinationContentId')};
+            }
+        },
+    },{
+        ATTRS: {
+            /**
+             * The destination content of the relation
+             *
+             * @attribute destinationContent
+             * @type Y.eZ.Content
+             */
+            destinationContent: {
+                value: null,
+            },
+            /**
+             * The destination content Id of the relation
+             *
+             * @attribute destinationContentId
+             * @type Integer
```

Comment:
it's not an integer, a REST id is a string, `/content/objects/42` for instance (and in any case `Integer` does not exist, it's `Number` in JavaScript)